api/v1beta1/openstackbaremetalset_types.go

```diff
@@ -56,6 +56,10 @@ type OpenStackBaremetalSetSpec struct {
 	// +kubebuilder:validation:Optional
 	// OSImage - OS qcow2 image Name
 	OSImage string `json:"osImage,omitempty"`
+	// +kubebuilder:default=sha256
+	// +kubebuilder:validation:Optional
+	// OSImageChecksumSuffix - The file extension suffix on the OSImage checksum
+	OSImageChecksumSuffix *string `json:"osImageChecksumSuffix,omitempty"`
```

Comment:
@abays  would we need a separate patch as well to dataplane-operator to set this to sha256sum? should we go ahead and merge this to get nfv unblocked?

Reply:
@slagle At this point I think we are leaning towards closing this PR for the reasons @rabi described.  Furthermore, the downstream EDPM image has been respun to use the old `sha256` checksum filename extension so now the OpenStack Baremetal operator should be working as-is.